### DISCUSSION TITLE
autoreset

### DISCUSSION THREAD

**Discussion by NiekN** [arendst/Tasmota#14155] [Q&A] [answered]
Is there a function in tasmota , that the switch is directly resetted after x sec when it is switched  ON ?

I have a CV reset button and also a doorbell  , that must be ON for max 1 sec and then turned off , even when doorbell button is still pressed

**Comment by ascillato** [comment, marked as answer]
See pulsetime configuration command in the docs. Thanks.

**Reply by NiekN** [reply]
ok thx , i will test that
